src/diem/testing/miniwallet/client.py

```diff
@@ -74,6 +75,9 @@ def send(self, method: str, path: str, data: Optional[str] = None) -> requests.R
             headers={k: v for k, v in headers.items() if v},
         )
         log_level = logging.DEBUG if resp.status_code < 300 else logging.ERROR
+        if self.events_api_is_optional and path.endswith("/events"):
+            log_level = logging.DEBUG
```

Comment:
If status code is >= 300, do we want to switch this from error to debug?

Reply:
> change log level to debug when called events api failed on the target wallet application, because the api is optional, when it is not implemented, logging an error causes confusion, thus logging a debug message is better.